modules/module/LoRAModule.py

```diff
@@ -129,6 +142,7 @@ def load_state_dict(self, state_dict: dict):
             self.prefix + ".lora_down.weight": state_dict.pop(self.prefix + ".lora_down.weight"),
             self.prefix + ".lora_up.weight": state_dict.pop(self.prefix + ".lora_up.weight"),
             self.prefix + ".alpha": state_dict.pop(self.prefix + ".alpha"),
+            self.prefix + ".dropout_pct": state_dict.pop(self.prefix + ".dropout_pct"),
```

Comment:
Here you are using the `.dropout_pct` suffix for the dropout probability. In other places it's `.dropout`. This should be consistent. Does it even make sense to save that value? It's only used during training, and it's not needed during inference. When training, the value can always be loaded from the TrainConfig object.

Reply:
Removed it here but, will resuming from a backup require this? That's my only concern.